### PR TITLE
Fix Glimmer 2 content normalization

### PR DESCRIPTION
**Pull request by chancancode**
This is the "Basic content rendering tests" item from #13127.

Also removed some FIXMEs and TODOs related to this issue and improved the test coverage.

/cc @rwjblue @chadhietala

### CODE REVIEW

File: packages/ember-glimmer/lib/helpers/if-unless.js [hunk 1]
```diff
@@ -28,9 +28,6 @@ function makeConditionalHelper(toBool) {
 
     if (toBool(condition)) {
       return positional.at(1).value();
-    } else {
-      // TODO: this should probably be `undefined`: https://github.com/emberjs/ember.js/pull/12920#discussion_r53213383
-      return '';
```
Comment (chancancode) [inline packages/ember-glimmer/lib/helpers/if-unless.js]: @GavinJoyce  ^

File: packages/ember-glimmer/lib/helpers/log.js [hunk 2]
```diff
@@ -21,7 +21,6 @@ import Logger from 'ember-metal/logger';
 */
 function log(params) {
   Logger.log.apply(null, params);
-  return ''; // FIXME: Helper will currently return and render 'undefined', remove when fixed
```
Comment (chancancode) [inline packages/ember-glimmer/lib/helpers/log.js]: ^ @green-arrow

File: packages/ember-glimmer/tests/integration/content-test.js [hunk 3]
```diff
@@ -58,59 +59,71 @@ moduleFor('Static content tests', class extends RenderingTest {
...
 
-    let text2 = this.assertTextNode(this.firstChild, 'hello');
+  ['@test it can render a dynamic path']() {
+    this.renderPath('message', { message: 'hello' });
 
-    this.assertSameNode(text1, text2);
+    this.assertText('hello');
 
-    this.runTask(() => set(this.context, 'message', 'goodbye'));
+    // FIXME: use @mmun's assertStableRerender
```
Comment (chancancode) [inline packages/ember-glimmer/tests/integration/content-test.js]: cc @mmun :P

**Comment by rwjblue** [comment]
LGTM

**Comment by chadhietala** [comment]
LGTM

File: packages/ember-glimmer/lib/helpers/if-unless.js (hunk 1 above)
Comment (GavinJoyce) [inline packages/ember-glimmer/lib/helpers/if-unless.js]: :yum:

File: packages/ember-glimmer/lib/helpers/log.js (hunk 2 above)
Comment (green-arrow) [inline packages/ember-glimmer/lib/helpers/log.js]: Woo! 😄

**Pull request merged**